intents/test_all.sh

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# https://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+#
+# Runs all the tests & exits if any of the test fails
```

Comment:
Maybe better to not exit as soon as any test fails, but to list down all the failing tests (with the reason) at the end.

Reply:
Yes listing down all failed tests will be better than only listing the first failed test.

Reply:
Done.

Reply:
Done